subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidStudioPerformanceTest.groovy

```diff
@@ -36,7 +36,7 @@ class RealLifeAndroidStudioPerformanceTest extends AbstractCrossVersionPerforman
 
     /**
      * To run this test locally you should have Android Studio installed in /Applications/Android Studio.*.app folder,
-     * or you should set "studioHome" system property with the Android Studio installation path,
+     * or you should set "studio.home" system property with the Android Studio installation path,
```

Comment:
The correct name of sysprop is `studio.home`. See https://github.com/gradle/gradle-profiler/blob/24731f1cde60599de426284236f1c428a3d18354/src/main/java/org/gradle/profiler/studio/tools/StudioFinder.java#L8

Reply:
Note that in the gradle/gradle build logic `studioHome` is used, see [here](https://github.com/gradle/gradle/blob/1f160de5f92212587214c85cbfa3ad363aa0858e/build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt#L130), maybe we should unify these two and change `studioHome` to `studio.home` too.

Reply:
@asodja Ah, I see. Or we could probably leave it as it was - `studioHome`, since it's kinda public thing, but usage of `gradle-profile`(alongside with it's properties) is an implementation detail, we haven't be consistent here necessarily.

So I'm going to revert this change back.